spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,20 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+If a comma is followed by a line end and then a right parenthesis, square
+bracket or curly brace (`)`, `]`, and `}`, respectively) then such a trailing
+comma is ignored. For example:
```

Comment:
If a comma (`,`) is followed immediately by a newline and a closing parenthesis (`)`), bracket (`]`), or brace (`}`), then the comma is treated as a "trailing comma" and is ignored.

Other specese doesn't say square or curly. The section on newlines uses "treated" in a casual sense. All these closing limiters are in the "parentheses" class defined earlier, but it's confusing to say "a parenthesis-class delimiter", etc.